src/FormField/Generic.php

```diff
@@ -78,7 +79,7 @@ public function init(): void
 
         if ($this->form && $this->field) {
             if (isset($this->form->fields[$this->field->short_name])) {
-                throw (new \atk4\ui\Exception('Form already has a field with the same name'))
+                throw (new Exception('Form already has a field with the same name'))
```

Comment:
This PR does not target these changes... But ok.

Reply:
I just fix where i see these if file is already open :)